x/mongo/driver/topology/connection.go

```diff
@@ -361,6 +364,15 @@ func (c initConnection) WriteWireMessage(ctx context.Context, wm []byte) error {
 func (c initConnection) ReadWireMessage(ctx context.Context, dst []byte) ([]byte, error) {
 	return c.readWireMessage(ctx, dst)
 }
+func (c initConnection) SetStreaming(streaming bool) {
+	c.currentlyStreaming = streaming
+}
+func (c initConnection) CurrentlyStreaming() bool {
+	return c.currentlyStreaming
+}
+func (c initConnection) SupportsStreaming() bool {
+	return c.canStream
```

Comment:
Is this ever set to true? I can't seem to find it anywhere...

Reply:
Not yet. This PR only adds support for streaming wire messages. This flag will be toggled in the next PR, which will allow us to stream `isMaster` responses. In that PR, we'll set `canStream` to true if the first `isMaster` response contains `topologyVersion` because that tells us we're talking to a 4.4+ server.